common_components/linux_compat/freertos/freertos_linux.c

```diff
@@ -145,6 +157,13 @@ QueueHandle_t xSemaphoreCreateRecursiveMutex(void)
 
 void vTaskDelete(TaskHandle_t *task)
 {
+    for (int i = 0; i < s_threads; ++i) {
+        if (task == s_notifiers->id) {
```

Comment:
The same question https://github.com/espressif/esp-protocols/pull/618#discussion_r1692587260

Reply:
good point, already resolved and fixed in https://github.com/espressif/esp-protocols/pull/618